spec/azure/virtual_network.go

```diff
@@ -1,7 +1,10 @@
 package azure
 
 type VirtualNetwork struct {
-	CIDR             string `json:"cidr" yaml:"cidr"`
-	MasterSubnetCIDR string `json:"masterSubnetCidr" yaml:"masterSubnetCidr"`
-	WorkerSubnetCIDR string `json:"workerSubnetCidr" yaml:"workerSubnetCidr"`
+	CIDR                    string `json:"cidr" yaml:"cidr"`
+	MasterSubnetCIDR        string `json:"masterSubnetCidr" yaml:"masterSubnetCidr"`
+	WorkerSubnetCIDR        string `json:"workerSubnetCidr" yaml:"workerSubnetCidr"`
+	APILoadBalancerCIDR     string `json:"apiLoadBalancerCidr" yaml:"apiLoadBalancerCidr"`
+	EtcdLoadBalancerCIDR    string `json:"etcdLoadBalancerCidr" yaml:"etcdLoadBalancerCidr"`
+	IngressLoadBalancerCIDR string `json:"ingressLoadBalancerCidr" yaml:"ingressLoadBalancerCidr"`
```

Comment:
From the user point of view I would like to do this. 

```
customObject.Spec.VitualNetwork.CIDR.Guest.Master
customObject.Spec.VitualNetwork.CIDR.Guest.Worker
customObject.Spec.VitualNetwork.CIDR.LB.API
customObject.Spec.VitualNetwork.CIDR.LB.Etcd
customObject.Spec.VitualNetwork.CIDR.LB.Ingress
```

I have no string feelings about the specific scheme and block names here. I just want to say that a more structured structure is more readable and easier to use for the actual user.

Reply:
Good point. I've restructured the LoadBalancer CIDRs to a separate block which I think makes it cleaner.